dasharo-compatibility/wifi-bluetooth-support.robot

```diff
@@ -73,7 +77,70 @@ WLE002.002 Wi-Fi scanning (Windows 11)
     Should Contain    ${out}    3mdeb_abr
     Should Contain    ${out}    3mdeb_abr_5GHz
 
-WLE003.001 Bluetooth scanning (Ubuntu 20.04)
+WLE003.001 Bluetooth scanning (Ubuntu 20.04) (Intel)
+    [Documentation]    Check whether the Bluetooth functionality of card is
+    ...    initialized correctly and can be used from within the
+    ...    operating system.
+    Skip If    not ${WIRELESS_CARD_BLUETOOTH_SUPPORT}    WLE003.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    WLE003.001 not supported
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    Scan For Bluetooth In Linux
+    Exit From Root User
+
+WLE004.001 Wireless card detection (Ubuntu 20.04) (Atheros)
```

Comment:
This is still a variant of the test:
`WLE001.00X Wireless card detection`
So the test number should remain the same, only the variant number should change.
```suggestion
WLE001.003 Wireless card detection (Ubuntu 20.04) (Atheros)
```

Reply:
Test case removed